.upstream-tests/test/std/utilities/time/time.clock/time.clock.system/local_time.types.pass.cpp

```diff
@@ -28,6 +28,7 @@
 
 int main(int, char**)
 {
+#ifndef __cuda_std__
```

Comment:
This effectively disables this test - why?

Reply:
because those types are guarded by `#ifndef __cuda_std__`

Reply:
Ah, because we haven't done time zones. Cool.